src/private_benches/secret_signed_ct.rs

```diff
@@ -0,0 +1,51 @@
+use crypto_bigint::{NonZero, RandomMod, U128, U4096};
+use dudect_bencher::{BenchRng, Class, CtRunner};
+use rand::Rng;
+
+use crate::{tools::Secret, uint::SecretSigned};
+/// Is `SecretSigned::new_positive` constant time when the bound is kept constant and the value varies?
+pub fn new_positive_with_constant_bound(runner: &mut CtRunner, rng: &mut BenchRng) {
+    let bound = 127;
+    let modulus = NonZero::new(U4096::ONE << bound).unwrap();
+    let input_len = 20_000_000;
+    let mut inputs = {
+        (0..input_len)
+            .map(|_| {
+                // "Normal" class: a random, typically big, value.
+                if rng.r#gen::<bool>() {
```

Comment:
Do the classes have to be interleaved randomly? Or can we generate a vector of one class and a vector of another and concatenate them?